community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/PlannerName.scala

```diff
@@ -68,21 +73,21 @@ case object DPPlannerName extends CostBasedPlannerName {
  * Rule based planner for classes of queries where the cost based planner might end up with suboptimal plans.
  */
 case object FallbackPlannerName extends CostBasedPlannerName {
-  val name = "FALLBACK"
+  val name = CostBasedPlannerName.default.name
```

Comment:
The name of `FallbackPlannerName` should be `FALLBACK` since it is a different thing wrt `IDP`.